yarn.lock

```diff
@@ -8337,6 +8337,11 @@ commander@3.0.0, commander@^3.0.0:
   resolved "https://registry.yarnpkg.com/commander/-/commander-3.0.0.tgz#0641ea00838c7a964627f04cddc336a2deddd60a"
   integrity sha512-pl3QrGOBa9RZaslQiqnnKX2J068wcQw7j9AIaBQ9/JEp5RY6je4jKTImg0Bd+rpoONSe7GUFSgkxLeo17m3Pow==
 
+commander@3.0.2:
```

Comment:
there's something a little goofy with the old resolution here, but probably yarn related
```
commander@3.0.0, commander@^3.0.0:
  version "3.0.0"
  resolved "https://registry.yarnpkg.com/commander/-/commander-3.0.0.tgz#0641ea00838c7a964627f04cddc336a2deddd60a"
  integrity sha512-pl3QrGOBa9RZaslQiqnnKX2J068wcQw7j9AIaBQ9/JEp5RY6je4jKTImg0Bd+rpoONSe7GUFSgkxLeo17m3Pow==

Reply:
ah i found it, can we update x-pack/commander to match?

Reply:
👍

Reply:
done